snorkel/mtl/loggers/checkpointer.py

```diff
@@ -6,24 +6,34 @@
 
 import torch
 
-from snorkel.types import Config
+from snorkel.mtl.snorkel_config import default_config
+from snorkel.mtl.utils import recursive_merge_dicts
 
 
 class Checkpointer(object):
     """Checkpointing Training logging class to log train infomation"""
 
-    def __init__(self, config: Config):
+    def __init__(self, **kwargs):
```

Comment:
What do we gain here by all of these `__init__(self, **kwargs)` vs. an `__init__` with properly types parameters, and collecting them into a dictionary? It seems like a convenience for developers at the expense of allowing users to pass in incorrect named or typed parameters. And without a docstring, I have no clue what could possible be passed in.

Reply:
See comment below. Discussed offline.